packages/backend/src/services/GithubAppService/GithubAppService.ts

```diff
@@ -160,9 +160,13 @@ export class GithubAppService extends BaseService {
         if (!isUserWithOrg(user)) {
             throw new Error('User is not part of an organization');
         }
+
+        // This endpoint is also used for developers on projects
+        // when using the sql runner, so we should allow access
+        // However github app is an organization property, so we can't check projects
         if (
             user.ability.cannot(
-                'update',
+                'view',
```

Comment:
if you have a better solution, let me know.

Reply:
hmmm good point - we can use `virtualView`? or is that too misleading?